package.json

```diff
@@ -19,6 +19,8 @@
   "main": "background.js",
   "dependencies": {
     "core-js": "^3.6.5",
+    "decompress-zip": "^0.2.2",
+    "dns": "^0.2.2",
```

Comment:
`fetch` doesn't seem to be available in the main process, so I'm using the `dns` library to check if the app can reach Github so it can download the template zips. `decompress-zip` is used to extract the content from the zipped archives that have the template files.

Reply:
I'm cool with `decompress-zip` but I'm wondering if we could use something built-in. Maybe electron's [net](https://www.electronjs.org/docs/api/net) module and make a HEAD request or something.

Reply:
I'll look into Electron's Net API and see if it's something that can be used instead.

Reply:
Fixed